spot-electron/src/config/config.js

```diff
@@ -0,0 +1,92 @@
+const { app } = require('electron');
+const fs = require('fs');
+const _ = require('lodash');
+const { logger } = require('../logger');
+const defaultConfig = require('../../config');
+
+const PERSISTED_CONFIG = `${app.getPath('userData')}/config.json`;
```

Comment:
Do you if this type of path accessing with forward slash also work on windows? (not saying change it now if needed)

Reply:
Good catch, I should have used node path